apollo-ios-codegen/Sources/ApolloCodegenLib/Templates/TemplateRenderer.swift

```diff
@@ -295,7 +295,7 @@ struct ImportStatementTemplate {
     static func template(for config: ApolloCodegen.ConfigurationContext) -> TemplateString {
       return """
       import ApolloTestSupport
-      import \(config.schemaModuleName)
+      \(if: config.output.schemaTypes.isInModule, "import \(config.schemaModuleName)")
```

Comment:
This might work for your particular configuration but it will break other configurations that use the `none` or `swiftPackage` options. The reason the schema module is always imported here is because it is intended to always be a separate module; [see my comment on your issue](https://github.com/apollographql/apollo-ios/issues/3262#issuecomment-1775676144).

Reply:
Got it, thanks for the explanation, will try to put the mock data in the test target!